app/Community/AuthServiceProvider.php

```diff
@@ -45,7 +45,7 @@ class AuthServiceProvider extends ServiceProvider
         ForumTopicComment::class => ForumTopicCommentPolicy::class,
         ForumTopic::class => ForumTopicPolicy::class,
         GameComment::class => GameCommentPolicy::class,
-        Message::class => MessagePolicy::class,
```

Comment:
I'd keep the MessagePolicy intact - we might still need it in the future.
E.g. for determining if a single message may be seen, edited, deleted.